### PR TITLE
JA: translated notebooks in site/ja/beta/tutorials/load_data

### PR DESCRIPTION
**Pull request by masa-ita**
Sorry to be late.

### CODE REVIEW

File: site/ja/tutorials/load_data/tf-records.ipynb [hunk 1]
```diff
@@ -0,0 +1,1687 @@
+{
```
Comment (lamberta) [inline site/ja/tutorials/load_data/tf-records.ipynb]: This file name should be: `site/ja/tutorials/load_data/tf_records.ipynb`

Comment (masa-ita) [reply site/ja/tutorials/load_data/tf-records.ipynb]: Thanks! I've just removed unnecessary one.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 2]
```diff
@@ -0,0 +1,932 @@
...
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "qX-DU34ZuKJX"
+   },
+   "outputs": [],
+   "source": [
+    "input_shape, output_shape = train_data.output_shapes\n",
+    "\n",
+    "input_dimension = input_shape.dims[1] # [0] is the batch size"
```
Comment (sfujiwara) [inline site/ja/beta/tutorials/load_data/csv.ipynb]: If we translate comments in source code, we have to translate it.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 3]
```diff
@@ -0,0 +1,932 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Wuqj601Qw0Ml"
+   },
+   "source": [
+    "## データのロード\n",
+    "\n",
+    "何をしているかはわかっていますが、扱っている CSV ファイルの先頭を見てみましょう。"
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thank you!
I've changed it to "それではいつものように".

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 4]
```diff
@@ -0,0 +1,932 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ZS-bt1LvWn2x"
+   },
+   "source": [
+    " データセットコンストラクタはこれらのラベルを自動的にピックアップします。\n",
+    "\n",
+    "使用するファイルの1行目にコラム名がない場合、`make_csv_dataset` 関数の `column_names` 引数に文字列のリストとして渡します。\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: I overlooked it.  Thanks.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 5]
```diff
@@ -0,0 +1,932 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "9cryz31lxs3e"
+   },
+   "source": [
+    "## データ処理"
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thank you.  Fixed.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 6]
```diff
@@ -0,0 +1,932 @@
...
+    "colab": {},
+    "colab_type": "code",
+    "id": "Co7UJ7gpNADC"
+   },
+   "outputs": [],
+   "source": [
+    "def get_dataset(file_path):\n",
+    "  dataset = tf.data.experimental.make_csv_dataset(\n",
+    "      file_path,\n",
+    "      batch_size=12, # Artificially small to make examples easier to show.\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thanks.  I overlooked it.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 7]
```diff
@@ -0,0 +1,932 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "To2qbBGGMO1D"
+   },
+   "source": [
+    "この処理を可視化するため、最初のバッチからカテゴリ列のレンソル1つを取り出し、処理を行い、前後の状態を示します。"
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thanks.  Fixed.

File: site/ja/beta/tutorials/load_data/csv.ipynb [hunk 8]
```diff
@@ -0,0 +1,932 @@
...
+    "  # 最初の ' ' を取り除く\n",
+    "  data = tf.strings.regex_replace(data, '^ ', '')\n",
+    "  # 最後の '.' を取り除く\n",
+    "  data = tf.strings.regex_replace(data, r'\\.$', '')\n",
+    "  \n",
+    "  # ワンホット・エンコーディング\n",
+    "  # data を1次元（リスト）から2次元（要素が1個のリストのリスト）にリシェープ\n",
+    "  data = tf.reshape(data, [-1, 1])\n",
+    "  # それぞれの要素について、カテゴリ数の長さの真偽値のリストで、\n",
+    "  # 真は要素がカテゴリラベルに一致していることを示すを作成\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thank you.  I've changed it.

File: site/ja/beta/tutorials/load_data/csv.ipynb (hunk 2 above)
Comment (masa-ita) [reply site/ja/beta/tutorials/load_data/csv.ipynb]: Thanks.  I've translated it as "[0] はバッチサイズ"

**Pull request merged**